### PR TITLE
Add translator comments

### PR DESCRIPTION
**Pull request by anyakro**
Add translator comments to all translatable strings, that describes whether the string is customer-facing or merchant-facing.

### CODE REVIEW

**Comment by anyakro** [comment]
@MichaelBengtsson I've gone through them all one last time, and I _think_ that they should all be correct except for three that are marked with `translators: [other]` because I'm unsure regarding how these three are used. There's obviously a correct answer for these though, perhaps we can look it over together?

File: klarna-payments-for-woocommerce.php [hunk 1]
```diff
@@ -375,8 +375,11 @@ public function plugin_action_links( $links ) {
 			$support_link = add_query_arg( 'subsection', 'support', $setting_link );
 
 			$plugin_links = array(
+				/* translators: [customer-facing]. */
 				'<a href="' . $setting_link . '">' . __( 'Settings', 'klarna-payments-for-woocommerce' ) . '</a>',
+				/* translators: [customer-facing]. */
 				'<a href="https://docs.woocommerce.com/document/klarna-payments/">' . __( 'Docs', 'klarna-payments-for-woocommerce' ) . '</a>',
+				/* translators: [customer-facing]. */
 				'<a href="' . $support_link . '">' . __( 'Support', 'klarna-payments-for-woocommerce' ) . '</a>',
```
[the remark this answers is not in the document]
Comment (MichaelBengtsson) [reply klarna-payments-for-woocommerce.php:383]: These should be merchant facing, since they are only shown in the admin plugin list.

File: classes/class-kp-assets.php [hunk 2]
```diff
@@ -261,6 +262,7 @@ public function enqueue_admin_script( $hook ) {
 		$klarna_payments_admin_params = array(
 			'get_unavailable_features'       => WC_AJAX::get_endpoint( 'kp_wc_get_unavailable_features' ),
 			'get_unavailable_features_nonce' => wp_create_nonce( 'kp_wc_get_unavailable_features' ),
+			/* translators: [customer-facing]. */
 			'select_all_countries_title'     => __( 'Select all', 'klarna-payments-for-woocommerce' ),
 		);
```
[the remark this answers is not in the document]
Comment (MichaelBengtsson) [reply classes/class-kp-assets.php:267]: This is a merchant facing string as well. So this should be changed.

File: classes/class-kp-ajax.php [hunk 3]
```diff
@@ -154,8 +154,10 @@ public static function kp_wc_auth_failed() {
 
 			$show_form = 'true' === $show_form ? true : false;
 			if ( $show_form ) {
+				/* translators: [customer-facing]. */
 				$order->add_order_note( __( 'Customer aborted purchase with Klarna.', 'klarna-payments-for-woocommerce' ) );
 			} else {
+				/* translators: [customer-facing]. */
 				$order->add_order_note( __( 'Authorization rejected by Klarna.', 'klarna-payments-for-woocommerce' ) );
 			}
```
[the remark this answers is not in the document]
Comment (MichaelBengtsson) [reply classes/class-kp-ajax.php:162]: These should be changed to merchant facing.

File: classes/class-wc-gateway-klarna-payments.php [hunk 4]
```diff
@@ -66,8 +66,10 @@ class WC_Gateway_Klarna_Payments extends WC_Payment_Gateway {
 	 * Constructor
 	 */
 	public function __construct() {
-		$this->id                 = 'klarna_payments';
-		$this->method_title       = __( 'Klarna for WooCommerce', 'klarna-payments-for-woocommerce' );
+		$this->id = 'klarna_payments';
+		/* translators: [customer-facing]. */
+		$this->method_title = __( 'Klarna for WooCommerce', 'klarna-payments-for-woocommerce' );
+		/* translators: [customer-facing]. */
 		$this->method_description = __( 'Supercharge your business with one single plugin for increased sales and enhanced shopping experiences.', 'klarna-payments-for-woocommerce' );
```
[the remark this answers is not in the document]
Comment (MichaelBengtsson) [reply classes/class-wc-gateway-klarna-payments.php:73]: This should be changed as well.

File: classes/class-wc-gateway-klarna-payments.php [hunk 5]
```diff
@@ -618,6 +624,7 @@ public function upsell( $order_id, $upsell_uuid ) {
 		$klarna_upsell_order = KP_WC()->api->upsell_klarna_order( $country, $klarna_order_id, $order_id );
 
 		if ( is_wp_error( $klarna_upsell_order ) ) {
+			/* translators: [other]. */
```
Comment (MichaelBengtsson) [inline classes/class-wc-gateway-klarna-payments.php]: This is a customer facing string.

File: classes/class-kp-klarna-express-checkout.php [hunk 6]
```diff
@@ -129,11 +129,13 @@ public function finalize_callback( $auth_token, $order_id, $order_key ) {
 
 		// Verify the order.
 		if ( ! $order ) {
+			/* translators: [other]. */
 			throw new Exception( __( 'Invalid order.', 'klarna-payments-for-woocommerce' ) ); // phpcs:ignore
 		}
 
 		// Verify the order key.
 		if ( $order->get_order_key() !== $order_key ) {
+			/* translators: [other]. */
```
Comment (MichaelBengtsson) [inline classes/class-kp-klarna-express-checkout.php]: These are both technically customer facing, but the error message here is not great for that honestly. But tag them as customer facing for now, and we should probably find a better string here for the message as well.

**Review by MichaelBengtsson — CHANGES_REQUESTED** [review]

File: klarna-payments-for-woocommerce.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (anyakro) [reply klarna-payments-for-woocommerce.php:383]: I've already changed these earlier, but for some reason it's not printing the "outdated" label!

File: classes/class-kp-assets.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (anyakro) [reply classes/class-kp-assets.php:267]: Same as above: I've already changed these earlier, but for some reason it's not printing the "outdated" label.

File: classes/class-kp-ajax.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (anyakro) [reply classes/class-kp-ajax.php:162]: Same as above: I've already changed these earlier, but for some reason it's not printing the "outdated" label.

File: classes/class-wc-gateway-klarna-payments.php (hunk 4 above)
[the remark this answers is not in the document]
Comment (anyakro) [reply classes/class-wc-gateway-klarna-payments.php:73]: Same as above.

File: classes/class-kp-klarna-express-checkout.php (hunk 6 above)
Comment (anyakro) [reply classes/class-kp-klarna-express-checkout.php]: Made a slight adjustment now, does that look OK or do we want to turn up the user-friendliness?

**Review by MichaelBengtsson — APPROVED** [review]
